autoload/arduino.vim

```diff
@@ -47,6 +47,11 @@ function! arduino#InitializeConfig() abort
   endif
   if !exists('g:arduino_serial_cmd')
     let g:arduino_serial_cmd = 'screen {port} {baud}'
+    if s:OS ==? 'Windows'
+      let g:arduino_serial_cmd = 'arduino-cli monitor -p {port} --config baudrate={baud}'
```

Comment:
If the arduino-cli supports reading the serial output now, we should just use this as the new default (provided `s:has_cli`)

Reply:
Yes, this is supported. In fact, I didn't know it wasn't supported before. I think it is a good idea to use it as default.